### DISCUSSION TITLE
IndexedDB to do example needs updating

### DISCUSSION THREAD

**Discussion by phewphewb** [mdn/mdn-community#615] [Content]
IndexedDB [to do](https://github.com/mdn/to-do-notifications) app example looks outdated. 
Would be good to update and refactor the example with the new ES6 syntax. 
A small start - [PR](https://github.com/mdn/to-do-notifications/pull/18).

**Comment by pepelsbey** [comment]
Closing as outdated/irrelevant.

**Reply by phewphewb** [reply]
yup it is 👍
